src/python/pants/backend/jvm/targets/jvm_target.py

```diff
@@ -90,6 +100,10 @@ def __init__(self,
       'strict_deps': PrimitiveField(strict_deps),
       'fatal_warnings': PrimitiveField(fatal_warnings),
       'zinc_file_manager': PrimitiveField(zinc_file_manager),
+      'javac_plugins': SetOfPrimitivesField(javac_plugins),
```

Comment:
I could imagine plugin ordering mattering in some cases, so rather than a set, this should probably be a list. And if it's a list, you can use PrimitiveField directly I think.